src/server/scripts/Outland/BlackTemple/boss_reliquary_of_souls.cpp

```diff
@@ -141,7 +141,10 @@ class boss_reliquary_of_souls : public CreatureScript
 
     struct boss_reliquary_of_soulsAI : public BossAI
     {
-        boss_reliquary_of_soulsAI(Creature* creature) : BossAI(creature, DATA_RELIQUARY_OF_SOULS), _inCombat(false) { }
+        boss_reliquary_of_soulsAI(Creature* creature) : BossAI(creature, DATA_RELIQUARY_OF_SOULS), _inCombat(false)
+        {
+            creature->m_SightDistance = 70.0f;
```

Comment:
already did... but original range that MoveInLineOfSight() start detect creatures is something like 50.0f...